local-antora-playbook.yml

```
@@ -15,7 +15,7 @@ content:
   - url: .
     branches: HEAD
   - url: https://github.com/redpanda-data/documentation
-    branches: [main, v/*, 'DOC-773-single-source-leader-pinning', api, shared, site-search]
+    branches: [main, v/*, 'DOC-780-Cloud-API-split-reference-into-control-plane-and-data-plane', shared, site-search]
```

```suggestion
    branches: [main, v/*, api, shared, site-search]
```
Change before merging